_search-plugins/knn/knn-index.md

```diff
@@ -90,8 +90,8 @@ Training data can be composed of either the same data that is going to be ingest
 ### Supported Lucene methods
 
 Method name | Requires training | Supported spaces | Description
-:--- | :--- | :--- | :---
-`hnsw` | false | l2, cosinesimil | Hierarchical proximity graph approach to Approximate k-NN search.
+:--- | :--- |:--------------------------------------------------------------------------------| :---
+`hnsw` | false | l2, cosinesimil, innerproduct (supported in OpenSearch version 2.13 and later) | Hierarchical proximity graph approach to Approximate k-NN search.
```

Comment:
It looks like we are inconsistent in our capitalization of "approximate" here: https://opensearch.org/docs/latest/search-plugins/knn/approximate-knn/.

Reply:
@natebower fixed all the places where there was issue in the file.

Reply:
Thank you, @navneet1v! Ready to merge.